hudi-common/src/main/java/org/apache/hudi/internal/schema/convert/InternalSchemaConverter.java

```diff
@@ -18,7 +18,10 @@
 
 package org.apache.hudi.internal.schema.convert;
 
-import org.apache.hudi.avro.AvroSchemaUtils;
+import org.apache.hudi.common.schema.HoodieJsonProperties;
+import org.apache.hudi.common.schema.HoodieSchema;
+import org.apache.hudi.common.schema.HoodieSchemaField;
+import org.apache.hudi.common.schema.HoodieSchemaType;
```

Comment:
I start to think that it's better to remove the `Hoodie` prefix from these class names in `org.apache.hudi.common.schema` packages, so it's easier to read, as eventually we are going to only maintain one schema system.  Wdyt?

Reply:
Sounds good.